server/apps/publish/content/tests.py

```diff
@@ -718,3 +719,20 @@ def test_added_removed_in_a_package(self):
         removed_items, added_items = ArchivePublishService()._get_changed_items(items, updates)
         self.assertEqual(len(removed_items), 1)
         self.assertEqual(len(added_items), 1)
+
+
+    def test_resolve_associations(self):
+      item = {
+        'associations': {
+          'sidebar': {'uri': '1'}
+        }
+      }
+      ArchivePublishService()._resolve_associations(item)
+      self.assertEqual('text', item['associations']['sidebar']['type'])
+
+      with self.assertRaises(ValidationError):
```

Comment:
Do we want to check for the correct error message as well, or is `_resolve_associations()` simply raising `ValidationError` already enough for our needs?